static/js/modules/helpers.js

```diff
@@ -117,6 +117,19 @@ Handlebars.registerHelper('electionUrl', function(year, options) {
   return new Handlebars.SafeString(url);
 });
 
+Handlebars.registerHelper('capitalize', function(value) {
```

Comment:
I would err on the side of removing it until it's needed again so it doesn't clutter up `helpers.js` if it doesn't ends up being used.